spec/syntax.md

```diff
@@ -77,52 +74,36 @@ The syntax specification takes into account the following design restrictions:
    It should be possible to define a message entirely on a single line with no ambiguity,
    as well as to format it over multiple lines for clarity.
 
-1. The syntax should not use nor reserve any keywords in any natural language,
-   such as `if`, `match`, or `let`.
-
 1. The syntax should define as few special characters and sigils as possible.
 
 ## Overview & Examples
 
 ### Simple Messages
 
-A simple message without any variables does not need any syntax:
+All messages, including simple ones, need `[…]` delimiters:
```

Comment:
I still think `{}` brackets are better than `[]` ones because of the need to quote inside the literal.

Reply:
Ack. The wording here is just being consistent with the rest of the doc in its current state. I know that #255 is still very much open.